src/react/components/window/Popup.jsx

```diff
@@ -135,8 +135,14 @@ export const Popup = ThemeConsumer(( {title = '', children, onClose, bringToTop,
     </div>
     */
 
-    const headerTheme = getHeaderTheme(theme);
-    return (<Container {...containerProps}>
+    let headerTheme;
+    if (options.customTheme) {
+        headerTheme = getHeaderTheme(options.customTheme);
```

Comment:
This feels a bit clunky, but right now I don't have a better suggestion how to do it. I'll have to think on it a bit more. At least headerTheme shouldn't be involved with the custom theme in my opinion for the style we are using here. Maybe just allow passing  a style-object directly instead? I'll try to think of a better way.